app/main.py

```diff
@@ -1,8 +1,56 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self
+                 , comfort_class: int
+                 , clean_mark: int
+                 , brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self
+                 , distance_from_city_center: float
+                 , clean_power: int
+                 , average_rating: float
+                 , count_of_ratings: float) -> None:
```

Comment:
The `count_of_ratings` parameter should be an integer, not a float, as it represents a count. Change the type from `float` to `int`.